src/widgets/copilot/components/Messages/error-message.scss

```
@@ -0,0 +1,48 @@
+  transition: transform 0.2s ease;
+}
+
+.gooey-error-details {
+  background-color: rgba(0, 0, 0, 0.04);
+  border-radius: 6px;
+
+  code {
+    font-size: 12px;
+    color: $dark-grey;
+    white-space: pre-wrap;
+    word-break: break-word;
```

Stylelint (17.7.0)

[error] Deprecated keyword "break-word" for property "word-break" (declaration-property-value-keyword-no-deprecated)

(declaration-property-value-keyword-no-deprecated)